package.json

```diff
@@ -44,36 +44,31 @@
     "babel-cli": "6.18.0",
     "babel-core": "6.20.0",
     "babel-eslint": "7.1.1",
+    "babel-jest": "^18.0.0",
```

Comment:
This is why I put `.npmrc` with `save-exact=true` in place near the beginning of the project. However,  since introducing yarn, it ignores `.npmrc`.  Does anyone know the equivalent in yarn?